README.md

```diff
@@ -1,2 +1,150 @@
-# portfolio-setup-and-mobile-first
-my personal portfolio's first sketches
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="images\testlogo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>ReadMe</b></h3>
+
+</div>
+
+### 👤 Author
+  
+**Nurbol Sultanov**
+
+- GitHub: [@everinurmind](https://github.com/everinurmind)
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Personal Portfolio <a name="about-project"></a>
+
+## 🛠 Built With HTML & CSS <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/">GitHub Desktop</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/">GitHub</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.microverse.org/"></a>Microverse</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- [production of main page] **[Main page of Personal Portfolio]**
+- [creating responsive page for portfolio] **[Mobile First Integration]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/everinurmind/portfolio-setup-and-mobile-first/tree/personal-portfolio)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to get access to repository.
+
+### Setup
+
+Clone this repository to your desired folder.
+
+### Install
+
+-
+
+### Run tests 
+
+To run tests, run the following command:
+
+- $ git status
```

Comment:
- [OPTIONAL] As we have appropriate testing commands in corresponding projects. you may consider adding this text here
`Tests have not applied yet`